src/Controls/samples/Controls.Sample.Sandbox/MauiProgram.cs

```diff
@@ -10,7 +10,9 @@ public static class MauiProgram
 		public static MauiApp CreateMauiApp() =>
 			MauiApp
 				.CreateBuilder()
+#if __ANDROID__ || __IOS__
 				.UseMauiMaps()
+#endif
```

Comment:
Is this intentional?

Reply:
yeah, you hit a exception on the sandbox , just copied from the other sample project